pkg/coscheduling/coscheduling.go

```diff
@@ -164,20 +164,19 @@ func (cs *Coscheduling) Permit(ctx context.Context, state *framework.CycleState,
 	}
 
 	namespace := p.Namespace
-	// TODO get actually scheduled(bind successfully) account from the SharedLister
-	running := cs.calculateRunningPods(podGroupName, namespace)
+	binded := cs.calculateBindedPods(podGroupName, namespace)
```

Comment:
Hmm... I don't quite think `bound` is better than `running`. Usually `running` means a Pod admitted by kubelet already, while `bound` just means the Pod's `.spec.nodeName` is already set.